lua/telescope/builtin/init.lua

```diff
@@ -35,6 +35,8 @@ builtin.git_branches = require('telescope.builtin.git').branches
 builtin.git_status = require('telescope.builtin.git').status
 
 builtin.builtin = require('telescope.builtin.internal').builtin
+builtin.menu = require('telescope.builtin.internal').menu
+builtin.test_menu = require('telescope.builtin.internal').test_menu
```

Comment:
We need a way to provide these tables. I know its currently just here for testing purposes but i have absolutely no idea how a user can define them in a good way. I am open for all suggestions.